levels/game-config.toml

```diff
@@ -2,7 +2,7 @@
 	title = "clone"
 	branch = "master"
 	solution_checker = "echo No pushing to master. Read the README file; exit 1"
-	flags = []  # ["start-here"], but it's implicit in the readme
+	flags = ["start-here"]  
```

Comment:
A bit of trailing whitespace at the end :)
https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces

Reply:
![](https://media.giphy.com/media/3o6ZthlLA0H2hwAfRK/giphy.gif)